app/validators/claim/advocate_hardship_claim_validator.rb

```diff
@@ -48,7 +48,7 @@ def validate_case_stage_id
   end
 
   def validate_offence
-    error_message = @record.agfs_reform? ? 'new_blank' : 'blank'
+    error_message = @record.agfs_reform? ? :new_blank : 'blank'
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/4632/files#r800874658)